apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -651,25 +650,36 @@ export const getStudentsMCSummaryForCurrentAssessment = state => {
     const studentsAssessment =
       studentsObject.responses_by_assessment[currentAssessmentId];
 
-    // If the student has not submitted this assessment, display empty results.
+    /* In order for the sorting of the submission timestamp column to work correctly in the SubmissionStatusAssessmentsTable, the submissionTimeStamp field must be a Date. So, we pass in arbitrary Dates in the past to handle when the assessment is in progress or not yet started. */
+    const inProgressFakeTimestamp = new Date('1990-01-01T20:52:05.000+00:00');
+    const notStartedFakeTimestamp = new Date('1980-01-01T20:52:05.000+00:00');
+
+    // If the student has not submitted this assessment
     if (!studentsAssessment) {
       return {
         id: studentId,
         name: studentsObject.student_name,
         isSubmitted: false,
-        submissionTimeStamp: i18n.notStarted()
+        inProgress: false,
+        submissionTimeStamp: notStartedFakeTimestamp
       };
     }
     // Transform that data into what we need for this particular table, in this case
     // it is the structure studentOverviewDataPropType
-    const submissionTimeStamp = studentsAssessment.submitted
-      ? new Date(studentsAssessment.timestamp).toLocaleString()
-      : i18n.inProgress();
+
+    /* In progress assessments have a timestamp from the server indicating when the student last worked on the assessment. We don't display that timestamp in the SubmissionStatusAssessmentsTable, but we use it here to check if the assessment has been started. */
+    const inProgress =
+      studentsAssessment.timestamp && !studentsAssessment.submitted;
+    const submissionTimeStamp = inProgress
+      ? inProgressFakeTimestamp
```

Comment:
i think instead of inserting a fake date here and on line 664, we could instead totally remove the timestamps from not started and in progress assessments (or set the timestamp to an empty string). this should allow us to still sort properly and will remove having to set up fake timestamps here

Reply:
If I don't set a date for the "in progress" and "not started" assessments, they jumble in the table. Also, empty strings can't be differentiated from each other in the sort. For example the sort might give me something like: not started, not started, in progress, not started, <timestamp>.

Reply:
ohh, yeah, you're right. i think the approach you have is the simplest/cleanest way to achieve what we want then!